frontend/kesaseteli/employer/src/components/application/companyInfo/CompanyInfoCell.tsx

```diff
@@ -1,20 +1,15 @@
 import { $CompanyInfoCell } from 'kesaseteli/employer/components/application/companyInfo/CompanyInfo.sc';
-import useApplicationApi from 'kesaseteli/employer/hooks/application/useApplicationApi';
+import useApplicationFormField from 'kesaseteli/employer/hooks/application/useApplicationFormField';
 import React from 'react';
-import LoadingSkeleton from 'react-loading-skeleton';
 import Company from 'shared/types/company';
 
 type Props = { field: keyof Company };
 
 const CompanyInfoCell: React.FC<Props> = ({ field }: Props) => {
-  const { applicationQuery } = useApplicationApi();
+  const { getValue } = useApplicationFormField<string>(`company.${field}`);
```

Comment:
another production code change. Reads the company value from react-query cache instead of react-query cache. Both should work same way in the application but in the tests only react-query cache works correctly.